server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InsertOperator.java

```diff
@@ -110,7 +110,7 @@ public PhysicalPlan generatePhysicalPlan(PhysicalGenerator generator)
                 measurementsNum, valueLists.get(i).length));
       }
       InsertRowPlan insertRowPlan =
-          new InsertRowPlan(device, times[i], measurementList, valueLists.get(i));
+          new InsertRowPlan(device, times[i], measurementList.clone(), valueLists.get(i));
```

Comment:
Does clone affect write performance?

Reply:
This is a quick-fix version... This will be fully refactored in the next version.